models/endpoints.model.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises");
+
+exports.selectAllEndpoints = () => {
+  return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((file) => {
+    return JSON.parse(file);
+  });
+};
```

Comment:
You don't really need a model for this one, models should just deal with the database. You could do this in the controller if you wanted to.